src/Microsoft.VisualStudio.Threading.Tests/Microsoft.VisualStudio.Threading.Tests.csproj

```diff
@@ -37,13 +33,13 @@
     <PackageReference Include="Xunit.Combinatorial" Version="1.2.7" />
     <PackageReference Include="xunit.runner.visualstudio" Version="2.4.1" />
     <PackageReference Include="Xunit.SkippableFact" Version="1.3.12" />
-    <PackageReference Include="Xunit.StaFact" Version="1.0.11-beta" />
+    <PackageReference Include="Xunit.StaFact" Version="0.3.18" />
```

Comment:
💭 Interesting change

Reply:
Works around https://github.com/dotnet/sdk/issues/3592 as described in https://github.com/AArnott/Xunit.StaFact/issues/35